CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ All notable, unreleased changes to this project will be documented in this file.
 - Add Sitemap generator - #342 by @bogdal
 - Add cypress tests - #333 by @AlicjaSzu
 - Add rich-text content renderer - #361 by @AlicjaSzu
+- Add rich-text content renderer for pages - #361 @ChanceLeachman
```

Comment:
it should be #426 instead of 361

Reply:
I have changed the issue number as stated.